ooni/nettests/experimental/tls_handshake.py

```diff
@@ -761,7 +785,7 @@ def handshakeSucceeded(connection):
             ## (which would be visible in pcaps anyway, since the FQDN is
             ## never encrypted) I do not see a way for this to log any user or
             ## identifying information. Correct me if I'm wrong.
-            self.report['session_key'] = session_key
+            self.report['session_key'] = {'b64': base64.standard_b64encode(session_key)} # json can't do binary
```

Comment:
It's probably good to use the same format we use for binary HTTP responses here. You can see how that is done here: https://github.com/TheTorProject/ooni-probe/blob/master/ooni/common/http_utils.py#L35.